electron/wine/runtimes/runtimes.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+import { existsSync, mkdirSync, unlinkSync, writeFileSync } from 'graceful-fs'
+import { join } from 'path'
+import { runtimePath } from './../../constants'
+import { logDebug, LogPrefix } from './../../logger/logger'
+import { Runtime, RuntimeName } from './../../types'
+import { extractTarFile, getAssetDataFromDownload } from './util'
+
+async function _get(): Promise<Runtime[]> {
+  mkdirSync(runtimePath, { recursive: true })
+  const allRuntimes = await axios.get('https://lutris.net/api/runtimes')
+  return allRuntimes.data
+}
+
+async function download(name: RuntimeName) {
+  const runtimes = await _get()
+  const runtime = runtimes.find((inst) => {
```

Comment:
Check if runtimes is not undefined, else you will get trace with `find is not a function of undefined`